.github/build-mandrel-images.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+PREFIX_NAME=quay.io/quarkus/ubi-quarkus-mandrel
```

Comment:
shuoldn't we refrain from using ubi-* prefix as this isn't a real ubi image ? 

p.s. I would check our other names but for the moment I can't access quay.io for some unknown reason ;/

Reply:
just checked the sources and we already use ubi-* thus never mind ;)